src/inspect_2.ado

```diff
@@ -11,13 +11,13 @@ Description
 
 A cross between {help summarize} and {help inspect}, __inspect_2__ gives the frequency of positive, zero, negative, and missing values in a variable, as well as the mean, minimum, and maximum value of a variable within those categories and overall.
 
-Users may specify the _save_ option to store the results in a matrix. The matrix (or matrices, if passing a {help varlist}) will be named _inspect_[varname]_.
+Results may be stored in a matrix (or matrices, if _varlist_ is multiple variables) by specifying matrix name(s) in the _matrix()_ option.
```

Comment:
The `save` option is still used in one of the examples below. Also, it seems like the command only works with `iweight` and `fweight`.

Also, the first example command should read `inspect_2 pincp pernp if agep >= 18 [fw=pwgtp]` to work on ACS. However, the command returns `no observations`. Seems to be because of missing observations for age, since this works: `inspect_2 pincp pernp if agep >= 18 & !missing(agep) [fw=pwgtp]`